src/plenoptic/simulate/canonical_computations/steerable_pyramid_freq.py

```diff
@@ -163,8 +180,11 @@ def __init__(self, image_shape, height='auto', order=3, twidth=1, is_complex=Fal
 
         # this list, used by coarse-to-fine optimization, gives all the
         # scales (including residuals) from coarse to fine
-        self.scales = (['residual_lowpass'] + list(range(self.num_scales))[::-1] +
-                       ['residual_highpass'])
+        self.scales = (
```

Comment:
They're different because `self.scales` is model-specific, and the Portilla-Simoncelli texture model additionally considers marginal pixel statistics (first several moments). However, `self.scales` is only used for coarse-to-fine optimization, and the pyramid generally should not be used as a model by itself (which is why it's in the `canonical_computations/` directory instead of `models/`). So I might remove the pyramid's scales attribute when I refactor it.

Reply:
It should definitely be defined in `PortillaSimoncelli` (though maybe `PortillaSimoncelli` could define its with `['pixel_statistics'] + self.pyr.scales` if I keep the pyramid's version), because it's necessary for coarse-to-fine optimization, which is necessary for that model.